pkg/cmd/openshift-sdn/network_config.go

```diff
@@ -0,0 +1,96 @@
+package openshift_sdn
```

Comment:
This file is moved over from pkg/cmd/server, but had enough deletions not to trigger the rename heuristic.